docs/develop/setup.md

```diff
@@ -80,88 +189,129 @@ can be found in `core_backend/add_users_to_db.py`.
+Ensure that you are in the `aaq-core` project directory and that you have activated the
+`aaq` virtual environment before proceeding.
 
-        export NEXT_PUBLIC_GOOGLE_LOGIN_CLIENT_ID=<YOUR_CLIENT_ID>
+---
 
-3. Run
+1. In a *new* terminal:
 
-        npm i
+        cd admin_app
+        nvm use 19
+        npm install
         npm run dev
 
-This will install the required packages required for the admin app and start the app in `dev` (autoreload) mode.
+    ??? note "Here's what you should see if the above commands execute successfully"
+            > admin-app@0.2.0 dev
+            > next dev
+
+            ▲ Next.js 14.1.3
+            - Local:        http://localhost:3000
 
-The admin app will now be accessible on [http://localhost:3000/](http://localhost:3000/)
+            ✓ Ready in 1233ms
+
+    This will install the required NodeJS packages for the admin app and start the
+    admin app in `dev` (i.e., autoreload) mode. The admin app will now be accessible on
+    [http://localhost:3000/](http://localhost:3000/).
+
+    You can login with either the default credentials
+    (username: `user1`, password: `fullaccess`) or the ones you specified
+    [when setting up the login credentials](#optional-login-credentials).
+
+2. To stop the admin app, exit the running app process in your terminal with
+`ctrl+c`.
 
 ## Set up docs
 
-1. [mkdocs](https://www.mkdocs.org/user-guide/installation/) should be installed in your
-development conda environment created by `make fresh-env`. Activate the conda environment:
+---
+**NB**
 
-        conda activate aaq
+Ensure that you are in the `aaq-core` project directory and that you have activated the
+`aaq` virtual environment before proceeding.
+
+---
 
-2. To host docs offline so you can see your changes, run the following in the root of the repo (with altered port so it doesn't interfere with the app's server):
+1. To host docs offline so that you can see documentation changes in real-time, run the
+following from `~/aaq-core` with an altered port (so that it doesn't interfere with the
+app's server):
 
-        mkdocs serve -a "localhost:8080"
+        mkdocs serve -a localhost:8080
 
 ## Pros and cons of each setup method
 
 | Method | Pros | Cons |
 | --- | --- | --- |
-| [Set up using docker compose watch](#set-up-using-docker-compose-watch) | <ul><li>Good for end-to-end testing</li><li>Local environment identical to production deployment</li><li>No need to setup local environment</li><li>Set environment variables and configs once</li></ul> | <ul><li>Changes take 5-10s to be reflected in the app</li></ul> |
+| [Set up using docker compose watch](#set-up-using-docker-compose-watch) | <ul><li>Good for end-to-end testing</li><li>Local environment identical to production deployment</li><li>No need to setup local environment</li><li>Set environment variables and configs once</li></ul> | <ul><li>Changes take 20-30s to be reflected in the app</li></ul> |
```

Comment:
With image size optimization, it now takes much longer (and quite impractical tbh :/ )